File: src/main/java/discordPanel/DiscordPanelImpl.java
package discordPanel;

import com.intellij.openapi.wm.ToolWindow;

import javax.swing.*;

public class DiscordPanelImpl {
    private final DiscordPanelViewer viewer;
    public DiscordPanelImpl(ToolWindow toolWindow) {
        viewer = new DiscordPanelViewer(toolWindow);
    }

    public JComponent getContent() {
        return viewer.getContent();
    }
}

File: src/main/kotlin/discordPanel/DiscordPanelImpl.kt
package discordPanel

import com.intellij.openapi.wm.ToolWindow
import javax.swing.JComponent

class DiscordPanelImpl(toolWindow: ToolWindow?) {
    private val viewer: DiscordPanelViewer = DiscordPanelViewer(toolWindow!!)
    val content: JComponent
        get() = viewer.content

}